pandas/_typing.py

```diff
@@ -91,7 +91,7 @@
 Suffixes = Tuple[str, str]
 Ordered = Optional[bool]
 JSONSerializable = Optional[Union[PythonScalar, List, Dict]]
-Axes = Collection
```

Comment:
do we actually use this?  grepping for ": Axes" im mostly (only?) seeing matplotlib types

Reply:
It's in the type signature for `DataFrame.__init__`

Reply:
we don't want to generically type like this